package/rules/rspack.js

```
@@ -0,0 +1,158 @@
+            }
+          }
+        }
+      }
+    }
+  ]
+})
+
+// CSS rules using Rspack's built-in CSS handling
+console.log("[Shakapacker] Checking for CSS loader...")
+if (moduleExists("css-loader")) {
+  console.log("[Shakapacker] css-loader found, loading CSS rule configuration...")
```

ESLint

[error] Replace `"[Shakapacker]·css-loader·found,·loading·CSS·rule·configuration..."` with `⏎····"[Shakapacker]·css-loader·found,·loading·CSS·rule·configuration..."⏎··`

(prettier/prettier)